spectator-web/src/main/java/com/netflix/spectator/controllers/model/ApplicationRegistry.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spectator.controllers.model;
+
+import com.netflix.spectator.controllers.model.MetricValuesMap;
+
+/**
+ * An ApplicationRegistry is the encoding of the metrics from a particular application.
+ *
+ * This is only public for testing purposes so implements equals but not hash.
```

Comment:
I could implement it, but then I'd need to figure out what an appropriate code is. If there's a way to do this, I'm all for it. I only implemented equals for testing.

If I make them package private classes, then I'd move them back into the controllers package, right? I'm ok with doing that. I moved them here to not muck things up. I need them for testing, but the tests can be in the same package. Should I do that?